tests/nextflow.config

```diff
@@ -1,5 +1,6 @@
-/*
-========================================================================================
-    Nextflow config file for running tests
-========================================================================================
-*/
+includeConfig 'https://raw.githubusercontent.com/nf-core/modules/0094ae45ba8f5a2e30644d4e252970775a03ee91/tests/config/test_data.config'
+
+timeline.enabled = false
+report.enabled   = false
+trace.enabled    = false
+dag.enabled      = false
```

Comment:
Why do we need this @adamrtalbot ?

Reply:
Break it down for me, yo!

Reply:
https://github.com/seqeralabs/nf-aggregate/actions/runs/7106051825/job/19344662333